ports/stm32/boards/NUCLEO_G474RE/board_init.c

```diff
@@ -0,0 +1,5 @@
+#include "py/mphal.h"
+
+void NUCLEO_G474RE_board_early_init(void) {
+
+}
```

Comment:
Unless there's a good reason to have this, since it's empty better to delete it (keeps things simpler).

Reply:
Right, I was questioning myself to keep it for 'just in case we need it later' . So now I removed it.